project/config/ppsni/config/views.view.slideshow.yml

```diff
@@ -172,8 +172,8 @@ display:
           exclude: false
           alter:
             alter_text: true
-            text: "{% if field_link_url is not empty %}\r\n<a href ={{ field_link_url }}>\r\n{% endif %}\r\n{{ field_photo }}\r\n{% if field_link_url is not empty %}\r\n</a>\r\n{% endif %}\r\n{% if field_teaser is not empty %}\r\n  <div class=\"views-field views-field-field-teaser\">\r\n    <div class=\"field-content slide__caption\">\r\n      {{ field_teaser }}\r\n    </div>\r\n  </div>\r\n{% endif %}"
-            make_link: false
+            text: "{% if field_link_url is not empty %}\r\n{% endif %}\r\n{{ field_photo }}\r\n{% if field_link_url is not empty %}\r\n{% endif %}\r\n{% if field_teaser is not empty %}\r\n  <div class=\"views-field views-field-field-teaser\">\r\n    <div class=\"field-content slide__caption\">\r\n      {{ field_teaser }}\r\n    </div>\r\n  </div>\r\n{% endif %}"
```

Comment:
I don't think you need this part `{% if field_link_url is not empty %}\r\n{% endif %}` anymore as there is nothing between the if statement? Same on NIJAC

Reply:
Makes sense, will update this! Thanks Adam